components/tools/OmeroJava/test/integration/AbstractServerTest.java

```diff
@@ -1306,6 +1330,9 @@ protected void assertNoneExist(String className, Collection<Long> ids) throws Se
     protected List<Pixels> importFile(OMEROMetadataStoreClient importer,
             File file, String format, boolean metadata, IObject target)
             throws Throwable {
+        if (importer == null) {
```

Comment:
Is this necessary? `createImporter()` will be called in line 1354 anyway.

Reply:
actually line 1354 ``importer`` should be used instead ``createImporter`` since an importer created by another class might be specified.
This was done while trying to narrow down the issue
@mtbc could you change it?

Reply:
I think it's likely I'll guess wrongly about what change is needed. Could you give me something to cherry-pick or just tell me exactly what which line should read instead and then I'll do it?

Reply:
Ohh, I think I understand now, I can do that.